dts/arm/nordic/nrf54h20_cpurad.dtsi

```diff
@@ -59,21 +59,36 @@ wdt011: &cpurad_wdt011 {};
 };
 
 &dppic130 {
-	owned-channels = <0>;
-	sink-channels = <0>;
-	nonsecure-channels = <0>;
+	owned-channels = <0 2 3>;
+	sink-channels = <0 2>;
+	source-channels = <3>;
+	nonsecure-channels = <0 2 3>;
 	status = "okay";
 };
 
 &dppic132 {
-	owned-channels = <0>;
-	source-channels = <0>;
-	nonsecure-channels = <0>;
+	owned-channels = <0 2 3>;
+	sink-channels = <3>;
+	source-channels = <0 2>;
+	nonsecure-channels = <0 2 3>;
 	status = "okay";
 };
 
 &ipct130 {
-	owned-channels = <0>;
-	source-channel-links = <0 3 0>;
+	status = "okay";
+	owned-channels = <0 2>;
+	sink-channel-links = <2 NRF_DOMAIN_ID_RADIOCORE 2>;
+	source-channel-links = <0 NRF_DOMAIN_ID_RADIOCORE 0
+					2 NRF_DOMAIN_ID_RADIOCORE 2>;
```

Comment:
This looks a bit weird. Why not split this into the units described in the binding: https://github.com/zephyrproject-rtos/zephyr/blob/3828c8b86ac5ab934014e740512d91764a0eaed7/dts/bindings/ipc/nordic%2Cnrf-ipct-common.yaml#L37-L39
IMHO this would be more readable:
```
	source-channel-links = <0 NRF_DOMAIN_ID_RADIOCORE 0>,
			       <2 NRF_DOMAIN_ID_RADIOCORE 2>;
```